fgrpc/grpcrunner_test.go

```diff
@@ -77,3 +77,53 @@ func TestGRPCRunner(t *testing.T) {
 		t.Errorf("Mismatch between requests %d and ok %v", totalReq, res.RetCodes)
 	}
 }
+
+func TestGRPCDestination(t *testing.T) {
+	tests := []struct {
+		name   string
+		dest   string
+		output string
+	}{
+		{
+			"hostname",
+			"localhost",
+			"localhost:8079",
+		},
+		{
+			"hostname and port",
+			"localhost:1234",
+			"localhost:1234",
+		},
+		{
+			"IPv4 address",
+			"1.2.3.4",
+			"1.2.3.4:8079",
+		},
+		{
+			"IPv4 address and port",
+			"1.2.3.4:5678",
+			"1.2.3.4:5678",
+		},
+		{
+			"IPv6 address",
+			"2001:dba::1",
+			"[2001:dba::1]:8079",
+		},
+		{
+			"IPv6 address and port",
+			"[2001:dba::1]:1234",
+			"[2001:dba::1]:1234",
+		},
```

Comment:
can you add those 2:

GRPCDestination(foo.bar.com:777) -> foo.bar.com:777
GRPCDestination(foo.bar.com) -> foo.bar.com:8079